src/main/java/com/forgeessentials/multiworld/command/CommandMultiworld.java

```diff
@@ -136,17 +137,26 @@ public static void parseCreate(CommandParserArgs arguments)
             }
             catch (NumberFormatException e)
             {
-                seed = arg.hashCode();
+                if (!arg.equals("-")) {
```

Comment:
Not sure how I like using `-` as a indicator for an empty seed but I think it is the best option.

Reply:
I agree. We can always change to something else if we find out a better indicator. :)

Sorry btw, I was not able to test this PR on my test server during the week, just were too busy at work. But I should be able to test this on Sunday or Monday for sure. :)